developers/weaviate/configuration/indexes.md

```diff
@@ -156,39 +161,41 @@ If we don't want to index the `Authors` we can simply skip all indices (vector _
     },
     "properties": [
         {
-            "indexInverted": false, // <== disable inverted index for this property
+            "indexFilterable": true,  // <== disable filterable index for this property
+            "indexSearchable": true,  // <== disable searchable index for this property
```

Comment:
`false` to disable, `true` (or simply skip) to enable. here value does not match comment. 
(also applies for following examples)

Reply:
Good catch, thank you